frontend/src/hooks/useAuthorization.ts

```diff
@@ -0,0 +1,50 @@
+import { useKeycloak } from "@react-keycloak/web";
+import { useEffect, useState } from "react";
+import { apiAxios } from "utils";
+
+const useAuthorization = (keycloak: { authenticated: boolean; tokenParsed: { email: string } }) => {
+  const [currentUser, setCurrentUser] = useState(undefined);
+
+  const handleCurrentUser = async () => {
+    if (keycloak.authenticated) {
+      if (!currentUser) {
+        await apiAxios()
+          .post(`users/email`, { email: keycloak.tokenParsed.email })
+          .then((user) => {
+            setCurrentUser(user.data.data);
+            return user.data.data;
+          })
+          .catch((err) => {
+            console.error("error:", err);
+          });
+      }
+    } else {
+      setCurrentUser(undefined);
+    }
+  };
+
+  useEffect(() => {
+    handleCurrentUser();
+  }, [useKeycloak().initialized]);
+
+  // const [currentUser, setCurrentUser] = useState(undefined);
```

Comment:
will this code be used in the future, or is this no longer required?  Either indicate future use or remove.